src/datepicker/datepicker-input.ts

```diff
@@ -263,7 +263,7 @@ export class NgbInputDatepicker implements OnChanges,
   constructor(
       private _parserFormatter: NgbDateParserFormatter, private _elRef: ElementRef<HTMLInputElement>,
       private _vcRef: ViewContainerRef, private _renderer: Renderer2, private _cfr: ComponentFactoryResolver,
-      private _ngZone: NgZone, private _service: NgbDatepickerService, private _calendar: NgbCalendar,
+      private _ngZone: NgZone, service: NgbDatepickerService, private _calendar: NgbCalendar,
```

Comment:
this change marks this as unused (`'service' is declared but its value is never read.ts(6133)`), should be probably just removed

Reply:
Good catch, @peterblazejewicz, thanks! Will update the PR